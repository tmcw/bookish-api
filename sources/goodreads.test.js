var GoodReads = require("./goodreads");

test("GoodReads", async () => {
  const gr = new GoodReads();
  expect(gr).toBeTruthy();
  expect(await gr.ISBN("0812993543")).toMatchSnapshot();
});
