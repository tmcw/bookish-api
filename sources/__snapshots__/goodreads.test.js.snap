// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoodReads 1`] = `
Object {
  "goodreads": Array [
    "25489625",
  ],
}
`;
